### PR TITLE
Added logging to file

### PR DESCRIPTION
**Pull request by kikimych**
Added file logging with rotation

### CODE REVIEW

File: VSRAD.DebugServer/VSRAD.DebugServer.csproj [hunk 1]
```diff
@@ -20,7 +23,7 @@
     <PropertyGroup>
        <Version>$([System.Text.RegularExpressions.Regex]::Match($([System.Text.RegularExpressions.Regex]::Match($([System.IO.File]::ReadAllText("$(MSBuildProjectDirectory)\..\VSRAD.Package\source.extension.vsixmanifest")), `Version=\"[0-9,.]+\" Language=`).ToString()),`[0-9,.]+`).ToString())</Version>
     </PropertyGroup>
-    <Message Importance="High" Text="Debug server version: $(Version)"/>
+    <Message Importance="High" Text="Debug server version: $(Version)" />
```
Comment (johnny-keker) [inline VSRAD.DebugServer/VSRAD.DebugServer.csproj:26]: I believe this change is accidental

**Comment by johnny-keker** [comment]
Small question - what behavior is expected when folder is read-only? As for now, server works as usual, but log file simply is not created. Do we want to show some warning in that case (and is it possible at all using this library)?

If library do not provide mechanisms for checking that situation, maybe we might want to create some `.tmp` to check if we have write permission in current directory?

File: VSRAD.DebugServer/Logging/GlobalLogger.cs [hunk 2]
```diff
@@ -16,19 +17,26 @@ static GlobalLogger()
             _executableName = Path.GetFileNameWithoutExtension(assembly.Location);
             _assemblyName = assembly.GetName().Name;
             _assemblyVersion = assembly.GetName().Version.ToString(3);
+            Log.Logger = new LoggerConfiguration()
+                .MinimumLevel.Information()
+                .WriteTo.Console()
+                .WriteTo.File("log.txt",
```
[the remark this answers is not in the document]
Comment (johnny-keker) [reply VSRAD.DebugServer/Logging/GlobalLogger.cs:23]: Actually log file name is something like `log202211.txt`, so library actually inserts year and month to filename. I would suggest just add somewhere there version of Debug Server as well and call it a day.

**Comment by kikimych** [comment]
> Small question - what behavior is expected when folder is read-only? As for now, server works as usual, but log file simply is not created. Do we want to show some warning in that case (and is it possible at all using this library)?
> 
> If library do not provide mechanisms for checking that situation, maybe we might want to create some `.tmp` to check if we have write permission in current directory?

It's impossible to control server file system from client side. And usually process running server should have credentials to write

**Comment by johnny-keker** [comment]
> > Small question - what behavior is expected when folder is read-only? As for now, server works as usual, but log file simply is not created. Do we want to show some warning in that case (and is it possible at all using this library)?
> > If library do not provide mechanisms for checking that situation, maybe we might want to create some `.tmp` to check if we have write permission in current directory?
> 
> It's impossible to control server file system from client side. And usually process running server should have credentials to write

I'm talking about server side only. I suggest server on startup check write permissions in current working directory. Project working dir is not the same as the dir when server lives, and if we want to store logs in server folder, than I guess it would be wise to check permissions for the user that runs this server instance, since it is not necessary the owner of this file

If we using standard library it seems strange to me, that it just ignores that problem and do not create log at all without any warning

**Comment by johnny-keker** [comment]
For example, we had some similar troubles with project config files, and some man-hours actually was spent to provide user with a clear warning that project dir is read-only, and even we provide mechanism to try to make it writable in the same dialog. That was solid requirement in that case, so I wander maybe we want something similar here?

File: VSRAD.DebugServer/Logging/GlobalLogger.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (kikimych) [reply VSRAD.DebugServer/Logging/GlobalLogger.cs]: Different instances of server will listen on different ports, but it is better to run them from separate directories

[the remark this answers is not in the document]
Comment (kikimych) [reply VSRAD.DebugServer/Logging/GlobalLogger.cs]: Renamed fileName

[the remark this answers is not in the document]
Comment (johnny-keker) [reply VSRAD.DebugServer/Logging/GlobalLogger.cs]: > Different instances of server will listen on different ports, but it is better to run them from separate directories

I'm not sure that this is the case. Why would I want to copy the entire server dir in order to run new instance of server?

File: VSRAD.DebugServer/VSRAD.DebugServer.csproj [hunk 3]
```diff
@@ -6,7 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Packaging.Tools.Trimming" Version="1.1.0-preview1-26619-01" />
+    <PackageReference Include="Microsoft.Packaging.Tools.Trimming" Version="1.1.0-preview1-26619-01"/>
```
Comment (johnny-keker) [inline VSRAD.DebugServer/VSRAD.DebugServer.csproj:9]: I was able to build fully functional Server in both Debug and Release configurations without this reference. I suppose it appeared here by accident

Comment (johnny-keker) [reply VSRAD.DebugServer/VSRAD.DebugServer.csproj:9]: Not 100% sure about this one, caching and building in VS studio is not straightforward by any means, but it looks like this package is not necessary 👀

Comment (johnny-keker) [reply VSRAD.DebugServer/VSRAD.DebugServer.csproj:9]: Oh, okay, looks like it were been for ages. Sorry, than it is off-topic regarding this PR, maybe we should revisit all the deps of the project someday 👀👀

**Review by johnny-keker — APPROVED** [review]
LGTM now!

**Pull request merged**